MiniBootcamp/MiniBootcamp/TweetModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  TweetModel.swift
+//  MiniBootcamp
+//
+//  Created by Chioma Amanda Mmegwa  on 29/05/2023.
+//
+
+import UIKit
+
+struct TweetCellViewModel {
+    let userName: String
```

Comment:
After declaring either a class, a struct or an enum, you have to leave and empty line.

```suggestion
struct TweetCellViewModel {

    let userName: String
```